### DISCUSSION TITLE
inscrieri ca voluntar

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#27] [FrontEnd Team]
Lasa-ne  o adresa de email sau un link de LinkedIN sau orice alta forma de contact sa putem lua legatura cu tine!
Ar fi util sa ne spui si ce vrei sa faci ca si voluntar la noi?

https://dev.to/sebiboga/peviitorro-voluntar-5hei

**Comment by GABRYI** [comment]
Buna ziua!
Sunt interesata de voluntariat ca tester manual si automat, nu am experienta, am terminat de curand un curs acreditat de testare manuala si automata.
 Las aici contul meu de LinkedIn:https://www.linkedin.com/in/costin-gabriela-1477a8227/
Multumesc!

**Reply by sebiboga** [reply]
linkul asociatiei:
https://www.oportunitatisicariere.ro/
